packages/gatsby-source-contentful/src/fetch.js

```diff
@@ -10,23 +10,63 @@ module.exports = async function contentfulFetch({
   reporter,
 }) {
   // Fetch articles.
+  let syncProgress
   const pageLimit = pluginConfig.get(`pageLimit`)
+  let syncItemCount = 0
```

Comment:
nit, putting const and let together

```suggestion
  const pageLimit = pluginConfig.get(`pageLimit`)
  let syncProgress
  let syncItemCount = 0
```